src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { FilterStatus } from '../../types/FilterStatus';
+import { TempTodo, Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+  filterStatus: FilterStatus;
+  tempTodo:TempTodo | null
+  handleDelete: (id: number) => void;
+  handleStatusChange: (todo: Todo) => void;
+  updatingTodoIds: number[];
+  editTodo: (todo: Todo, newTitle:string) => void;
+};
+
+export const TodoList:React.FC<Props> = ({
+  todos,
+  filterStatus,
+  tempTodo,
+  handleDelete,
+  handleStatusChange,
+  updatingTodoIds,
+  editTodo,
+}) => {
+  const visibleTodos = todos.filter(todo => {
```

Comment:
better to do it in `App.tsx` and pass `visibleTodos` as `todos` to the `TodoList` component
In this case, `TodoList` won't know anything about the filter